apps/api-gateway/schema.graphql

```diff
@@ -36,16 +36,13 @@ type Query {
   _(where: transactionFilter, orderBy: String, orderDirection: String, before: String, after: String, limit: Int): transactionPage!
   tokenPrice(timestamp: BigInt!): tokenPrice
   tokenPrices(where: tokenPriceFilter, orderBy: String, orderDirection: String, before: String, after: String, limit: Int): tokenPricePage!
-  historicalTreasury(date: BigInt!): historicalTreasury
```

Comment:
Since we will not use the "total treasury" and "Dao token's value" from the provider, it does not make sense to have a table and a persistent state anymore. So, im removing this table.